### ISSUE TITLE
[BUG] Failed to perform commissioning step 17

### ISSUE THREAD

**Issue by Agatha2333** [project-chip/connectedhomeip#26309]
### Reproduction steps

Hello, when I commission my nano strip. encountered the problem like : 
[1682919609.222886][9686:9688] CHIP:CTL: Failed to perform commissioning step 17
[1682919609.222895][9686:9688] CHIP:CTL: Going from commissioning step 'ThreadNetworkEnable' with lastErr = '../../src/app/CommandSender.cpp:248: CHIP Error 0x00000032: Timeout' -> 'Cleanup'

I tried to factory reset my otbr by sudo ot-ctl command but it does not work

### Bug prevalence

every time

### GitHub hash of the SDK that was being used

linux

### Platform

other

### Anything else?

[NETWORK.txt](https://github.com/project-chip/connectedhomeip/files/11363749/NETWORK.txt)

**Comment by bzbarsky-apple** [comment]
@Agatha2333 What that log shows is that chip-tool sent the EnableNetwork command and the server never responded.  Eventually the command invoke timed out.

What does the server log say?

**Comment by bzbarsky-apple** [comment]
No, I mean the log from the "nano strip" being commissioned.  How you do that, exactly, depends on the specific device involved.

**Comment by bzbarsky-apple** [comment]
I mean that the issue is that the device never responded with success (or failure, or anything else) to the command to join the Thread network.  The log from the client gives us no way to figure out why it didn't respond.  Did the device crash?  Is it just buggy and does not implement that command correctly?  Did it simply take longer to join the network than the timeout the client used? Something else?

So if you want to figure out why this is not working, we need to understand what the device is doing when it receives this command, and that almost certainly requires the device log.  You _might_ be able to make some conclusions based on the OTBR log (e.g. whether the device tried to join the network at all), but that will not tell you anything about why the device never responded.

**Comment by varun2103** [comment]
@Agatha2333 I also got same error, could you please how you resolved it ?

**Comment by varun2103** [comment]
@Agatha2333 I also got the same error could you please say me how you resolved it ?

**Comment by varun2103** [comment]
I factory reset the device but still I'm getting same error
